crates/conductor_lib/src/config.rs

```diff
@@ -707,6 +707,41 @@ pub enum StorageConfiguration {
     Pickle { path: String },
 }
 
+impl StorageConfiguration {
+    fn is_memory(&self) -> bool {
+        match self {
+            StorageConfiguration::Memory => true,
```

Comment:
You could probably just self == StorageConfiguration::Memory here

Reply:
Thanks for the tip, will do! Currently I'm just trying to sort out getting next to persistently work across terminal sessions so that I can test locally.